internal/static/integrations/cloudwatch_exporter/docs/doc.go

```diff
@@ -61,7 +61,7 @@ func generateServicesDocSection() string {
 	var sb strings.Builder
 	for _, supportedSvc := range yaceConf.SupportedServices {
 		sb.WriteString(
-			fmt.Sprintf("- Namespace: `%s` or Alias: `%s`\n", supportedSvc.Namespace, supportedSvc.Alias),
```

Comment:
You will need to run this to update the service list in the docs. ATM they all still have the Alias.